ReactApp/src/misc/Helpers.js

```diff
@@ -37,6 +37,22 @@ export function showBlockPicker() {
 	}
 }
 
+export function getPost() {
+	const currentPost = window?.initialPost;
```

Comment:
I envisioned all globals residing under a single `GBKit` namespace to avoid potential collisions. 

```js
const { initialPost: currentPost } = getGBKit();
```

Do you feel that is valuable?

Reply:
Let's try it! It'd be a good idea to have all of the editor params in the same namespace/localStorage 👍

Reply:
Updated in https://github.com/wordpress-mobile/GutenbergKit/pull/21/commits/54aedfb4541fb8465a6b4caa2c2c58f1d86334a3